packages/block-library/src/image/image-editing/cropper.js

```diff
@@ -48,7 +48,7 @@ export default function ImageCropper( {
 				'is-applying': isInProgress,
 			} ) }
 			style={ {
-				width,
+				width: width || clientWidth,
```

Comment:
I notice this is the only place `width` is even referenced. Only `clientWidth` is used elsewhere (lines 39 and 42) which makes it look like only using it here would be fine too, ie:
```suggestion
				width: clientWidth,
```

Reply:
This should be set to `width` when that is available, which is by default available.